tests/templates/kuttl/ldap-authentication/authcheck.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import sys
+import logging
+
+
+def main():
+    result = 0
+
+    proto = "http"
+    druid_cluster_name = "derby-druid"
+
+    druid_ports = {
+        "coordinator": 8081,
+        # "broker": 8082,
+        # "middlemanager": 8091,
+        # "historical": 8083,
+        # "router": 8888
+    }
+    log_level = 'INFO'
+    logging.basicConfig(level=log_level, format='%(asctime)s %(levelname)s: %(message)s', stream=sys.stdout)
+
+    for role, port in druid_ports.items():
```

Comment:
Will we ever need this loop in the future if we only connect to the coordinator?

Reply:
The authcheck in the authorizer test connects to every single role, instead of just one. Should we expand the list here as well?

Reply:
As it's a quick check, and we are thorough in similar test cases, I'll add all roles to the loop here.